code_of_conduct.md

```diff
@@ -59,8 +59,13 @@ representative at an online or offline event.
 ## Enforcement
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported to the community leaders responsible for enforcement at
-https://www.research.net/r/S9SF5NP.
+reported to the community leaders responsible for enforcement, who are the
+ffmprovisr maintainers:
+* Ashley Blewer (ashley.blewer@gmail.com)
```

Comment:
This change is fine for me in terms of having this particular email address for myself  here - would want to confirm everyone else was ok with theirs being public before any merging though.